packages/core/src/sass/account-wizard.scss

```diff
@@ -106,7 +107,22 @@
     height: calc(100% - 57px);
     position: relative;
     width: 100%;
+    @include desktop {
+        overflow: auto;
```

Comment:
Why `overflow: auto` here? This should be handled by a `ThemedScrollbars`. We don't want to have un-styled scrollbars on Desktop.